### PR TITLE
chore: 2.6.0 release proposal

### PR DESCRIPTION
**Pull request by kjin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,94 @@
...
 
+## 2018-03-30, Version 2.6.0 (Beta), @kjin
+
+This version introduces non-null spans, a new plugin loading mechanism, revised log messages and changes to `pg` tracing support (now including version 7).
+
+### Notable Changes
+
+#### Non-Null Spans
+
+`SpanData` object passed by `traceApi.runInRootSpan` and returned by `traceApi.createChildSpan` are _guaranteed_ to be non-null, whereas they previously could be null if either (1) they weren't created because of tracing policy decisions (an __untraced__ span) or (2) an attempt to create the span was made in a place where it was impossible to determine on behalf of which incoming request they were tracing (an __uncorrelated__ span). In other words:
```
Comment (DominicKramer) [inline CHANGELOG.md]: nit: object -> objects

**Comment by kjin** [comment]
I will merge this and release 2.6.0 on Monday, 4/2.

**Comment by d-winter** [comment]
Hey @kjin Will 2.6.0 be released today? Would be awesome since we would love to upgrade:)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,94 @@
...
+
+#### Plugin Loader
+
+The plugin loading mechanism has been completely re-written. There should be no observable changes (please file an [issue](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/issues) if you encounter one.) The re-write fixes an issue where plugins can't patch modules with circular dependencies correctly ([#618](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/issues/618)).
+
+#### Tracing `pg`
+
+We now support tracing for `pg` versions 6 - 7.
+
+For consistency between tracing version 6 and 7, span labels in `pg` (when `config.enhancedDatabaseReporting` is enabled) now contain pre-processed query values rather than post-processed values.
```
Comment (ofrobots) [inline CHANGELOG.md]: Side observation (should not affect this PR): would it make sense to have plugin specific documentation pages with information like this?

**Comment by kjin** [comment]
@d-winter Yes! I will release it imminently

File: CHANGELOG.md (hunk 2 above)
Comment (kjin) [reply CHANGELOG.md]: Good idea, I've opened #709 to track this.

**Pull request merged**